int32 degrees
---
bool succes